Source/WebCore/svg/SVGGraphicsElement.cpp

```diff
@@ -132,11 +132,15 @@ AffineTransform* SVGGraphicsElement::ensureSupplementalTransform()
 
 void SVGGraphicsElement::attributeChanged(const QualifiedName& name, const AtomString& oldValue, const AtomString& newValue, AttributeModificationReason attributeModificationReason)
 {
-    SVGTests::parseAttribute(name, newValue);
-    SVGElement::attributeChanged(name, oldValue, newValue, attributeModificationReason);
-
+    // Updating the SVG transform baseVal needs to happen before calling attributeChanged().
+    // attributeChanged() invokes svgAttributeChanged(), which calls RenderLayerModelObjects
+    // repaintOrRelayoutAfterSVGTransformChange() method, if LBSE is activated. To determine
+    // if the SVG transform changed, m_transform has to be updated before.
     if (name == SVGNames::transformAttr)
         m_transform->baseVal()->parse(newValue);
+
+    SVGTests::parseAttribute(name, newValue);
+    SVGElement::attributeChanged(name, oldValue, newValue, attributeModificationReason);
```

Comment:
I looked at [262844@main](https://commits.webkit.org/262844@main) and after [264763@main](https://commits.webkit.org/264763@main) and [262985@main](https://commits.webkit.org/262985@main), I found these places have still the order flipped:

1. SVGAElement::attributeChanged
2. SVGAnimateMotionElement::attributeChanged
3. SVGAnimateTransformElement::attributeChanged
4. SVGAnimationElement::attributeChanged
5. SVGClipPathElement::attributeChanged
6. SVGFEImageElement::attributeChanged
7. SVGFEMergeNodeElement::attributeChanged
8. SVGFEOffsetElement::attributeChanged
9. SVGFETileElement::attributeChanged
10. SVGFontFaceElement::attributeChanged
11. SVGFontFaceUriElement::attributeChanged
12. SVGGeometryElement::attributeChanged
13. SVGGlyphRefElement::attributeChanged
14. SVGGradientElement::attributeChanged
15. SVGGraphicsElement::attributeChanged
16. SVGImageElement::attributeChanged
17. SVGMarkerElement::attributeChanged
18. SVGMaskElement::attributeChanged
19. SVGPathElement::attributeChanged
20. SVGPatternElement::attributeChanged
21. SVGPolyElement::attributeChanged
22. SVGSVGElement::attributeChanged
23. SVGStopElement::attributeChanged
24. SVGStyleElement::attributeChanged

I think the order in most of these places need to be reverted to the original order before [262844@main](https://commits.webkit.org/262844@main)

Reply:
I re-inspected all classes touched in Chris original patch as well and additionally found:
25. SVGFEComponentTransferElement::attributeChanged